src/features/recurringInvoices/RecurringInvoices.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+
+import PageContent from '../../common/pageContent/PageContent';
+import PageTitle from '../../common/pageTitle/PageTitle';
+import DataSummary, { DataSummaryProps } from '../../common/dataSummary/DataSummary';
+import Button from '../../common/button/Button';
+import Table, { Column, COLUMN_WIDTH } from '../../common/table/Table';
+import Text from '../../common/text/Text';
+import InternalLink from '../../common/internalLink/InternalLink';
+import Section from '../../common/section/Section';
+import Pagination from '../../common/pagination/Pagination';
+import styles from './recurringInvoices.module.scss';
+
+export interface FailedInvoices {
+  id: string;
+  customerId: string;
+  customerName: string;
+  harbor: string;
+  failureReason: string;
+}
+
+export interface RecurringInvoicesProps {
+  loading: boolean;
+  dataSummary: DataSummaryProps['labelValuePairs'];
+  failedInvoicesCount: number | undefined;
+  failedInvoicesData: FailedInvoices[];
+  handleSend(dueDate: string | null): void;
+}
+
+const RecurringInvoices = ({
+  loading,
+  dataSummary,
+  failedInvoicesCount,
+  failedInvoicesData,
+  handleSend,
+}: RecurringInvoicesProps) => {
+  const { t } = useTranslation();
+
+  const failedInvoicesCols: Column<FailedInvoices>[] = [
```

Comment:
We might actually want a separate column for the date when the failure occurred. But that should be a separate task.

Reply:
That makes sense, but there is no such field exposed from the backend, is there?